src/component/manage_my_tariffs.js

```diff
@@ -18,8 +18,17 @@ export default class ManageMyTariffs extends ViewBase{
     this.filterText = "";
   }
 
+	noTariffsBannerTemplate() {
+		return html`<div class="w3-panel w3-round w3-light-grey">
+			<p>${this.t("manageMyTariffsBanner")}</p>
+			<a href="https://www.mybestchargingcard.com/" target="_blank" class="w3-btn pc-secondary w3-margin-top w3-margin-bottom">Check out now!</a>
```

Comment:
`Check out now` should also be translated.

Reply:
Fixed.